app/views/TaxYears.scala.html

```diff
@@ -14,16 +14,17 @@
  * limitations under the License.
  *@
 
-@import views.html.templates.main_template
+@import views.html.templates.MainTemplate
 @import models.calculation.CalculationResponseModelWithYear
 @import views.html.helpers._
 @import uk.gov.hmrc.play.views.html.helpers.form
 @import views.html.helpers.backLinkHelper
 
+@this(mainTemplate: MainTemplate, appConfig: config.FrontendAppConfig)
```

Comment:
Please replace the imported helpers with their injectable versions:
```suggestion
@import views.html.helpers.injected.BackLink

@this(mainTemplate: MainTemplate, backLinkHelper: BackLink, appConfig: config.FrontendAppConfig)
```